performance/src/performance/timer.cpp

```diff
@@ -19,10 +24,13 @@ namespace traccc::performance {
 timer::timer(const std::string_view timer_name, timing_info& t_info)
     : m_start(std::chrono::high_resolution_clock::now()),
       m_name(timer_name),
-      m_timing_info(t_info) {}
+      m_timing_info(t_info) {
+      nvtxRangePushA(timer_name.data());
```

Comment:
The problem that we're discussing in Mattermost is coming from here. You have to surround the `nvtxRangePushA(...)` and `nvtxRangePop()` function calls with

```c++
#ifdef TRACCC_HAVE_NVTX
...
#endif
```

statements.

Reply:
I used clang-format to do the formatting. It passed on my folded repository.